synapse/push/pusherpool.py

```diff
@@ -346,17 +346,22 @@ async def _start_pusher(self, pusher_config: PusherConfig) -> Optional[Pusher]:
             )
             return None
 
-        if not p:
+        if not pusher:
             return None
 
         appid_pushkey = "%s:%s" % (pusher_config.app_id, pusher_config.pushkey)
 
         byuser = self.pushers.setdefault(pusher_config.user_name, {})
         if appid_pushkey in byuser:
```

Comment:
Would it be clearer to use `pusher.app_id` or `appid_pushkey` consistently? Are those the same?

Reply:
Huh, I wonder why `pushers` isn't just a `Dict[str, Dict[Tuple[str, str], Pusher]`, but whatever that's getting into too much refactoring.